tests/karaf-client-integration-tests/pom.xml

```diff
@@ -36,6 +36,18 @@
    </properties>
 
    <dependencies>
+      <dependency>
+         <groupId>org.apache.activemq</groupId>
+         <artifactId>activemq-client</artifactId>
+         <scope>test</scope>
+         <exclusions>
+            <exclusion>
+               <!--Let's compile against JMS 2-->
```

Comment:
The comment says "JMS 2," but the artifactId references the 1.1 spec. This seems wrong.

Reply:
Sorry about that. I didn't notice the exclusion. Makes perfect sense.